### PR TITLE
Introduction to Graph

### PR DESCRIPTION
**Pull request by Pradeep-Kurapati**
Hey! I thought it is important to have an overview of what a Graph so anyone who visited can make sense of Bellman Ford algorithm. Please let me know if there need to be any changes. Thank you.

### CODE REVIEW

**Comment by Panquesito7** [comment]
Hello! Do you need any help with this? 🙂

**Comment by Pradeep-Kurapati** [comment]
Hey! I didn't know that deleting this repository in my profile would close the pull request.
What I did is I added a brief definition of graphs in data structures folder

File: en/Data Structures/Graph/Graph.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (appgurueu) [inline en/Data Structures/Graph/Graph.md:1]: Leading newline

File: en/Data Structures/Graph/Graph.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# Graph
+
+A Graph is a non-linear data structure consisting of vertices and edges. The vertices are sometimes also referred to as nodes and the edges are lines or arcs that connect any two nodes in the graph. More formally a Graph is composed of a set of vertices( V ) and a set of edges( E ). The graph is denoted by G(E, V).
+## Components of a Graph
+
+1. **Vertices**: Vertices are the fundamental units of the graph. Sometimes, vertices are also known as vertex or nodes. Every node/vertex can be labeled or unlabelled.
+
+2. **Edges**: Edges are drawn or used to connect two nodes of the graph. It can be ordered pair of nodes in a directed graph. Edges can connect any two nodes in any possible way. Sometimes, edges are also known as arcs.
+# Source
```
Comment (appgurueu) [inline en/Data Structures/Graph/Graph.md:10]: Please add a newline above headings

File: en/Data Structures/Graph/Graph.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+
+# Graph
+
+A Graph is a non-linear data structure consisting of vertices and edges. The vertices are sometimes also referred to as nodes and the edges are lines or arcs that connect any two nodes in the graph. More formally a Graph is composed of a set of vertices( V ) and a set of edges( E ). The graph is denoted by G(E, V).
```
Comment (appgurueu) [inline en/Data Structures/Graph/Graph.md:4]: `vertices (V)`* & `edges (E)`*

File: en/Data Structures/Graph/Graph.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+## Components of a Graph
+
+1. **Vertices**: Vertices are the fundamental units of the graph. Sometimes, vertices are also known as vertex or nodes. Every node/vertex can be labeled or unlabelled.
+
+2. **Edges**: Edges are drawn or used to connect two nodes of the graph. It can be ordered pair of nodes in a directed graph. Edges can connect any two nodes in any possible way. Sometimes, edges are also known as arcs.
```
Comment (appgurueu) [inline en/Data Structures/Graph/Graph.md:9]: ordered pair (v1, v2) with v1, v2 in V; set `{v1, v2}` in an undirected graph (may be just `{v1}` if a node is connected to itself)

**Pull request closed without merging**